wire-runtime/src/commonMain/kotlin/com/squareup/wire/internal/Internal.kt

```diff
@@ -209,7 +209,7 @@ fun sanitize(values: List<String>): String {
 }
 
 fun boxedOneOfClassName(oneOfName: String): String {
-  return oneOfName.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() }
```

Comment:
btw that is what the kotlin lib would `replaceWith` if one uses `capitalize()`

Reply:
Yeah it has to maintain the old behavior and its quirks so that the replacement works unconditionally. Here we're safe to always transform the first character though.